### PR TITLE
feat: improve english translations

### PR DESCRIPTION
**Pull request by azolus**
The language data inside i18n tags has been reviewed. Changes  include:
 - Correcting grammatical and spelling errors
 - Consistent use of tenses
 - Reducing the amount of abbreviations like "you'll" or "he's"
 - Small stylistic tweaks to improve consistencies across languages
chore: resolve rebase conflict with vue3/all branch

fix: rename directories to avoid merge conflict

### CODE REVIEW

File: nuxt/components/event/list/EventList.vue [hunk 1]
```diff
@@ -81,5 +81,5 @@ watch(eventsQuery.error, (currentValue, _oldValue) => {
 de:
   noEvents: Aktuell gibt es keine Veranstaltungen 😕
 en:
-  noEvents: There are currently no events 😕
+  noEvents: Nothing going here... 😕
```
Comment (dargmuesli) [inline nuxt/components/event/list/EventList.vue:84]: Let's keep some sort of reference to `events`.

File: nuxt/components/form/account/FormAccountSignIn.vue [hunk 2]
```diff
@@ -171,11 +171,11 @@ de:
   username: Nutzername
   verificationMailResend: Verifizierungsmail erneut senden
 en:
-  jwtStoreFail: Failed to store the authentication information!
+  jwtStoreFail: Failed to store the authentication data!
   passwordReset: Reset password
-  postgres22023: An account with this username does not exists! Check your input for spelling mistakes.
-  postgres55000: The email address is not yet verified!
-  postgresP0002: Login failed! Have you registered yet? Check your input for spelling mistakes.
+  postgres22023: This username does not belong to any account! Check your input for spelling errors.
```
Comment (dargmuesli) [inline nuxt/components/form/account/FormAccountSignIn.vue:176]: I'd like to keep `mistakes` as `error` sounds harsh to me.

File: nuxt/components/form/account/password/FormAccountPasswordReset.vue [hunk 3]
```diff
@@ -118,8 +118,8 @@ en:
   accountPasswordReset: Reset password
   accountPasswordResetSuccess: Password reset successfully.
   passwordNew: New Password
-  postgres22023: The password is too short! Think about a longer one.
-  postgresP0002: Unknown reset code! Have you perhaps already reset your password?
-  postgres55000: The reset code has expired!
+  postgres22023: This password is too short! Think of a longer one.
+  postgresP0002: Invalid reset code! Have you perhaps already reset your password?
+  postgres55000: This reset code has expired!
```
Comment (dargmuesli) [inline nuxt/components/form/account/password/FormAccountPasswordReset.vue:123]: Maybe mention the `link` instead.

File: nuxt/components/form/account/password/FormAccountPasswordResetRequest.vue [hunk 4]
```diff
@@ -117,9 +117,9 @@ de:
   requestAccepted: Anfrage angenommen!
 en:
   accountPasswordResetRequest: Send email
-  accountPasswordResetRequestSuccess: "Set a new password using the link that you can find in the email that you'll receive shortly."
+  accountPasswordResetRequestSuccess: Choose a new password using the verification link provided by email.
```
Comment (dargmuesli) [inline nuxt/components/form/account/password/FormAccountPasswordResetRequest.vue:120]: `sent to you`?

File: nuxt/components/form/event/FormEvent.vue [hunk 5]
```diff
@@ -552,15 +552,15 @@ en:
   isRemote: remote
   maximumInviteeCount: Maximum guest count
   name: Name
-  namePlaceholder: Welcome Party
+  namePlaceholder: Slug Party
```
Comment (dargmuesli) [inline nuxt/components/form/event/FormEvent.vue:555]: ?

File: nuxt/components/form/event/FormEvent.vue [hunk 6]
```diff
@@ -552,15 +552,15 @@ en:
...
   location: Location
   preview: Preview
   previewNoContent: No content to preview 😕
   slug: Slug
   slugPlaceholder: welcome-party
   start: Start
   updated: Updated
-  validationExistenceNone: You've already created an event with id "{slug}"
+  validationExistenceNone: You have already created an event with id "{slug}"
```
Comment (dargmuesli) [inline nuxt/components/form/event/FormEvent.vue:563]: Do you want to keep it this way?

File: nuxt/pages/dashboard/index.vue [hunk 7]
```diff
@@ -112,12 +112,12 @@ en:
   contactsMine: My contacts
   dashboard: dashboard
   eventsMine: My events
-  eventsMineDescription: Here you will soon see your events.
+  eventsMineDescription: Here, you will soon see your events.
```
Comment (dargmuesli) [inline nuxt/pages/dashboard/index.vue:115]: Maybe without comma.

File: nuxt/pages/index.vue [hunk 8]
```diff
@@ -251,15 +251,15 @@ en:
   maevsiDescription: maevsi is the manager for events supported by invitees.
   maevsiDescription1: Personal invitations. Proper feedback.
   maevsiDescription2: Don't miss out on events your friends are going to.
-  smartphoneDescription: With meavsi you no longer have to organize your events via chat groups anymore. You communicate all important information clearly, keep control over individual invitations and have your head free for the content of your event – your guests will thank you for it.
-  smartphoneTitle: Do you still organize your event like that?
-  stepCreateDescription: Piece of mind for you and your guests, instead of a bustling group chat
+  smartphoneDescription: With meavsi, you no longer have to organize your events via chat groups anymore. You communicate all important informations clearly, keep control over individual invitations and focus on the content of your event without distractions – your guests will thank you for it.
```
Comment (dargmuesli) [inline nuxt/pages/index.vue:254]: Let's keep `information` and `distraction`

File: nuxt/pages/index.vue [hunk 9]
```diff
@@ -251,15 +251,15 @@ en:
...
+  smartphoneTitle: You still organize your event like that?
```
Comment (dargmuesli) [inline nuxt/pages/index.vue:255]: Revert

File: nuxt/pages/index.vue [hunk 10]
```diff
@@ -251,15 +251,15 @@ en:
...
+  stepCreateDescription: Peace of mind for you and your guests, instead of a bustling group chat
   stepCreateNumber: '1'
   stepCreateTitle: Create an event
   stepDefineDescription: all acceptances and declines under your control
   stepDefineNumber: '3'
   stepDefineTitle: Receive feedback
-  stepInviteDescription: achieve attention through personal invitation texts
+  stepInviteDescription: pique interest through personal invitation texts
```
Comment (dargmuesli) [inline nuxt/pages/index.vue:262]: `gather`

File: nuxt/pages/event/[username]/[event_name]/index.vue [hunk 11]
```diff
@@ -595,12 +595,12 @@ en:
...
   events: events
-  feedbackRequest: 'Please provide feedback whether you will be attending:'
+  feedbackRequest: 'Please confirm if you will attend:'
   greeting: Hey{usernameString}!
   greetingDescription: "You've been invited to the following event."
   hintQrCode: This picture is your access authorization for the event
-  iCalDownload: Download as calendar item
-  iCalHint: You can open the downloaded file with your calendar app then.
+  iCalDownload: Download for your calendar
+  iCalHint: You can open the download with your calendar app to load it.
```
Comment (dargmuesli) [inline nuxt/pages/event/[username]/[event_name]/index.vue:603]: What happens in the calendar app is not important to us.

**Review by dargmuesli — CHANGES_REQUESTED** [review]
`spelling error` is used in multiple files

File: nuxt/components/form/event/FormEvent.vue (hunk 6 above)
Comment (azolus) [reply nuxt/components/form/event/FormEvent.vue:563]: In this specific case "You've already..." sounds too informal to me

File: nuxt/pages/index.vue (hunk 9 above)
Comment (azolus) [reply nuxt/pages/index.vue:255]: Revert what exactly? 255 makes sense. What about 254 and 256?

Comment (dargmuesli) [reply nuxt/pages/index.vue:255]: `Do you`

**Review by dargmuesli — APPROVED** [review]

**Comment by dargmuesli** [comment]
Thank you! :green_heart:

**Pull request merged**